### ISSUE TITLE
The results of mini2CUB are opposite

### ISSUE THREAD

**Issue by lampardwk** [bl0/negative-margin.few-shot#12]
Sorry to bother you again!I reproduced the experiment on my own device and found that most of the results were basically the same. However, for the mini2CUB experiment, I found that the experimental results of neg-softmax and neg-cosine were just opposite to those in the paper.The experimental result of neg-softmax is about 67%, while that of neg-cosine is about 69%.

**Comment by bl0** [comment]
Hi, thanks a lot for your attention to our work. Can you send your log and trained models to my email (liubinthss@gmail.com)? I will check the difference between your experiments with ours.

**Comment by lampardwk** [comment]
Thank you for your reply! Due to the size limit, I could not send the email, so I saved it in the network disk:
Liink：https://pan.baidu.com/s/1xXvVFgObnKtgCFratxezrQ 
Password：6666

**Comment by bl0** [comment]
Hi, I found the log that we rerun after the code refactors and release to GitHub, and found that the neg-cosine indeed can achieve the accuracy of 69%, which is higher than the paper reported 67%. While neg-cosine achieves 68.79%, which is slightly worse than the paper reported 69%, but in a proper range.

![image](https://user-images.githubusercontent.com/9164121/103331835-a8146b80-4aa2-11eb-9787-dd17485c04db.png)

BTW, the variance of accuracy for the mini2CUB and CUB is large and you can run the test episode longer, such as 6000 episodes, to get a more confident accuracy. (I run for 600 episodes because we have a lot of experiments to run, and 600 episode is the standard setting.)

**Comment by bl0** [comment]
PS: I also checked the accuracy in the raw log before the paper submission, and it is the same as the paper.

**Comment by lampardwk** [comment]
OK, Thank you very much for your advice.
